### DISCUSSION TITLE
disable "Cancel button"

### DISCUSSION THREAD

**Discussion by Mo-alali** [grahampugh/erase-install#345] [Q&A] [answered]
Hi,

is it possible to disable "Cancel button" 

I have tried disable "button1disabled" is work and disabled continue

Thank you

**Comment by grahampugh** [comment]
At which stage? Download, Confirm, Upgrade, or Erase?

**Reply by Mo-alali** [reply]
Confirm

**Reply by grahampugh** [reply]
No, it's not possible. The `--confirm` flag is optional for when you want to give the user the chance to cancel. If you don't want to give them that option, don't use the `--confirm` flag.

**Reply by Mo-alali** [reply]
and Erase?
I don't want to give user the chance to cancel it!

**Reply by grahampugh** [reply]
There is not a Cancel button on the Erase screen.

**Reply by mickl089** [reply]
But there is a cancel button during the user data input :-)

**Reply by grahampugh** [reply]
I guess you're talking about the password input screen? (Not the confirmation screen). If so, see my answer below.

**Comment by PhillyPhoto** [comment]
How can we get rid of the cancel button on the password prompt on ARM chip machines? We're looking to use this to enforce upgrades when MDM pushes fail and other fringe cases that we need to patch for zero day vulnerabilities, but letting people just cancel the whole thing removes the enforceability.

![Screenshot 2023-06-01 at 3 20 46 PM](https://github.com/grahampugh/erase-install/assets/18701195/734cd259-6e7a-4b1e-a58d-45936af6bf5a)

**Reply by grahampugh** [reply, marked as answer]
Use --max-password-attempts=infinite to prevent canceling the password dialog.
